packages/graphql/src/schema.ts

```diff
@@ -18,19 +20,35 @@ process.cwd ??= () => ''
 
 const isCodegen = Boolean(process.env.CYPRESS_INTERNAL_NEXUS_CODEGEN)
 
+const types = [entities, constants, customScalars, dirname ? null : testingTypes]
```

Comment:
What is this `dirname` conditional doing?

Reply:
`dirname` doesn't exist in Vite, so it's saying that if we don't have it we include the testing union types on the exposed schema.

Reply:
Could probably name this better to clarify though